hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -701,6 +701,27 @@
   </description>
 </property>
 
+  <property>
+    <name>hadoop.security.token.service.use_ip</name>
+    <value>true</value>
+    <description>
+      Controls whether tokens always use IP addresses.
+      DNS changes will not be detected if this option is enabled.
+      Existing client connections that break will always reconnect
+      to the IP of the original host. New clients will connect
+      to the host's new IP but fail to locate a token.
+      Disabling this option will allow existing and new clients
+      to detect an IP change and continue to locate the new host's token.
+
+      In secure multi-homed environments, the following parameter
```

Comment:
"the following parameter" should be "this parameter" if it means hadoop.security.token.service.use_ip?